tests/mocha/event_test.js

```diff
@@ -538,8 +543,12 @@ suite('Events', function() {
         suite('fromJson', function() {
           testSuite.testCases.forEach((testCase) => {
             test(testCase.title, function() {
-              var event = new testCase.class(...testCase.getArgs(this));
-              var event2 = new testCase.class();
+              // Some classes that are optional requires need to be dynamically
+              // loaded via goog.module.get.
+              const testClass = testCase.classModule ?
+                  goog.module.get(testCase.classModule) : testCase.class;
+              var event = new testClass(...testCase.getArgs(this));
+              var event2 = new testClass();
```

Comment:
So this has caused a bit of an adventure haha

I originally thought this should not be needed so I commented it out and all my tests passed. I then looked into it more and I am now confused why commenting out those lines did not break everything. Is this fixing an error that you were getting? If yes, what was the error? I *think* we might be able to do this by including the appropriate files in the `mocha/index.html` file instead of having to do this bit, but I'm not entirely sure.

Reply:
Sorry for sending you down that path! When I wrote this the test was failing without these changes, but it now appears that the test passes as it was, presumably since fixing up the dependencies in other files in the meantime has sorted things out here. Reverted the test file to its original state.